lib/Hakyll/Web/Tags.hs

```diff
@@ -143,6 +150,11 @@ buildTags :: MonadMetadata m => Pattern -> (String -> Identifier) -> m Tags
 buildTags = buildTagsWith getTags
 
 
+--------------------------------------------------------------------------------
```

Comment:
Sorry, this is such a nitpick, but I think this function should be dropped. If we keep it, a question arises why there are no `…ByField` versions of other functions that call `getTags`, like `tagsField`. I fear that if we go down that path, we'll end up with an exponential growth.

It's easy enough to implement `buildTagsByField` and `tagsFieldByField` on the user side, so let them do it I think.

Reply:
No worries, that' a fair point. I had a similar concern but had already implemented this one. 

It's removed.